fat-jar.html

```diff
@@ -0,0 +1,206 @@
+<h2>Creating a fat Jar</h2>
+
+<div class="alert alert-warning">
+<p>
+    <strong>Warning</strong>: Since Java 9, applications should be modular, and distributed
+    with tools like <a href="index.html#modular">jlink</a>. However, if you still need to have a 
+    non modular application and you want to distribute a single jar, you can do a fat jar.
+</p>
+</div>
+<p>
+    You can create a runnable fat jar for your JavaFX 11 project on
+    <a href="#Maven">Maven</a>, <a href="#Gradle">Gradle</a> or even 
+    <a href="#command-line">command line</a>.
+</p>
+
+<p>
+    As explained <a href="http://mail.openjdk.java.net/pipermail/openjfx-dev/2018-June/021977.html">here</a>,
+    in order to create a runnable jar with all the required JavaFX dependencies, 
+    you will need to use a launcher class that doesn't extend from <kbd>Application</kbd>.
+</p>
+<p>
+    For the purpose of this guide, we'll use a <a href="hellofx/src/main/java/hellofx/Launcher.java">Launcher</a> class, 
+    that calls to the <kbd>main</kbd> method of the <a href="hellofx/src/main/java/hellofx/HelloFX.java">HelloFX</a> class. 
+    You can download a simple <kbd>hellofx</kbd> project from <a href="hellofx.zip">here</a>.
+</p>
+
+<a name="Maven"></a><h3>Maven</h3>
+
+<p>
+    If you develop your JavaFX applications using Maven, you don't have to download
+    the JavaFX SDK. Just specify the modules and the versions you want in the <kbd>pom.xml</kbd> file,
+    and the build system will download the required modules, including the native libraries 
+    for your platform.
+</p>
+<p>
+    Here is a <a href="hellofx/pom.xml">pom.xml</a> file which shows how to create a 
+    runnable fat jar for the <kbd>hellofx</kbd> project with the Maven shade plugin.
+</p>
+
+<p>
+    Run the plugin to create the fat jar:
+</p>
+
+<pre><code>
+mvn compile package
+</code></pre>
+
+<p>
+    Run the application using:
+</p>
+
+<pre><code>
+java -jar shade/hellofx.jar
+</code></pre>
+
+<h4>Cross-platfrom jar</h4>
+
+<p>
+    You can create a cross-platform fat jar by adding the dependencies from the 
+    three platforms that require native libraries to your pom file, in this sample 
+    only the ones from <kbd>javafx.graphics</kbd>:
+</p>
+
+<textarea rows="22" cols="80" style="border:none;">
```

Comment:
Why a TextArea? This makes the code editable.

Reply:
I looked for a solution to render xml easily. You are right about being editable. I'll make it non editable, non resizable.